vortex-array/src/array/chunked/canonical.rs

```diff
@@ -88,9 +89,11 @@ pub(crate) fn try_canonicalize_chunks(
             )))
         }
 
-        // TODO(aduffy): better list support
         DType::List(..) => {
-            todo!()
+            // TODO: improve performance
```

Comment:
Make sure you use the `TODO(username):` pattern so we can ask the author even modulo new git commits

Reply:
But also why is this comment here? This function is fast! It only calls another function

Reply:
I think using a list view in the future would help avoid the offset rebase